aiidalab_widgets_base/process.py

```diff
@@ -412,8 +412,12 @@ def update(self):
 
         output_file_path = None
         if 'remote_folder' in self.calculation.outputs:
-            output_file_path = os.path.join(self.calculation.outputs.remote_folder.get_remote_path(),
-                                            self.calculation.attributes['output_filename'])
+            try:
+                output_file_path = os.path.join(self.calculation.outputs.remote_folder.get_remote_path(),
+                                                self.calculation.attributes['output_filename'])
+            except KeyError:
+                self.placeholder = f"The `output_filename` attribute isn't set for {self.calculation.process_class}. " \
+                "The output won't appear here."
```

Comment:
```suggestion
                self.placeholder = f"The `output_filename` attribute is not set for {self.calculation.process_class}. " \
                "Nothing to show."
```
We should generally avoid contractions like "isn't" or "won't" in formal messages IMO. I also found the second sentence slightly awkward.